### ISSUE TITLE
unclear error message

### ISSUE THREAD

**Issue by GoogleCodeExporter** [uzh/gc3pie#273]
```
I just spotted this error message in the VM-MAD Orchestrator output;
the message comes from the `gc3.gc3libs` logger:

    ERROR: Could not update job information for the last 120 seconds, try [1]. Consider setting to Unknown state.

I cannot really understand the last half of it:

  - what does "try [1]" mean? should I try what?
  - who should "consider setting to unknown state"? what should be set to unknown?

Bottom line: I think I know the answers for all the above, but I can
see how it would be confusing for a newbie user ;-)



```

Original issue reported on code.google.com by `riccardo.murri@gmail.com` on 18 Jul 2012 at 9:25
